olp-cpp-sdk-dataservice-read/src/repositories/DataRepository.h

```diff
@@ -29,10 +29,16 @@ namespace olp {
 namespace dataservice {
 namespace read {
 class DataRequest;
+class TileRequest;
 namespace repository {
 
 class DataRepository final {
  public:
+  static DataResponse GetVersionedDataTileQuadTree(
```

Comment:
```suggestion
  static DataResponse GetVersionedTile(
```
How we do it internally is our business. Doesn't need to be extra marked in API.

Reply:
My point is we don't need to add implementation details to the method name. Just call it `GetVersionedTile`. How we get the tile or the metadata internally is not important for the API name.

Reply:
Renamed